ChillChaser/Models/Request/CreateAppUsage.cs

```diff
@@ -4,6 +4,7 @@ public class CreateAppUsage
     {
         public required DateTime From { get; set; }
         public required DateTime To { get; set; }
+        public required  AppSession[] Sessions {get; set;}
```

Comment:
Use `IEnumerable<AppSession>`

Reply:
also is required necessary for this?

Reply:
Tænker det er nemmest hvis den er required så man skal sætte den

Reply:
It is done og synes required er good